src/demo/DemoTestOrganizationService.java

```diff
@@ -207,14 +216,15 @@ public static void testGetCentresByPage() throws MambuApiException {
 
 	public static void testGetCentresByBranch() throws MambuApiException {
 
-		System.out.println(methodName = "\nIn testGetCentresByBranch");
+		methodName = new Object() {}.getClass().getEnclosingMethod().getName();
+		System.out.println(methodName = "\nIn " + methodName);
 		OrganizationService organizationService = MambuAPIFactory.getOrganizationService();
 
 		String branchId = BRANCH_ID;
 		String offset = "0";
 		String limit = "500";
-		System.out.println("\nIn testGetCentresByBranch" + "  BranchID=" + branchId + "  Offset=" + offset + "  Limit="
-				+ limit);
+		System.out.println(
+				"\nIn testGetCentresByBranch" + "  BranchID=" + branchId + "  Offset=" + offset + "  Limit=" + limit);
```

Comment:
It's very nice that you used the new logging approach to all of the tests. I think we can use it also here. What do you think?